Packs/CarbonBlackEnterpriseEDR/Playbooks/Carbon_Black_EDR_Search_Process.yml

```diff
@@ -0,0 +1,234 @@
+id: Carbon Black EDR Search Process
+version: -1
+name: Carbon Black EDR Search Process
+starttaskid: "0"
+description: |-
+  Use this playbook as a sub-playbook to run a process search task.
+  This playbook implements polling by continuously running the `cb-eedr-process-search-results` command until the operation completes.
+  The remote action should have the following structure:
+
+  1. Initiate the operation - insert the arguments for te search command.
+  2. Poll to check if the operation completed.
+  3. Get the results of the operation.
+tasks:
+  "0":
```

Comment:
let's add a task to make sure we have a live instance, as we have seen in the detonate PBs